modules/ROOT/pages/configure-openshift-routes.adoc

```diff
@@ -0,0 +1,99 @@
+= Configuring Red Hat OpenShift Routes in Runtime Fabric
+
+Runtime Fabric enables you to configure Red Hat OpenShift Routes to create ingress endpoints for your Mule applications. 
+
+== Before You Begin
+
+Configuration of routes in Runtime Fabric follows closely to ingress resource configuration. Before you create routes, review the following:
+* xref:custom-ingress-configuration.adoc#how-ingress-resources-templates-work-in-runtime-fabric[How Ingress Resources Templates Work in Runtime Fabric] 
+* xref:custom-ingress-configuration.adoc#example-ingress-resource-templates[Example Ingress Resource Templates]
+
+This documentation also assumes you’re familiar with https://docs.openshift.com/container-platform/4.10/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration[Creating a route through an Ingress object^].
```

Comment:
Would it be better if we just add this item in the bullet list as another requisite? That way it looks more consistent and that topic seems to be a prerequisite as well.

Reply:
Yeah, good call!